src/components/SearchByToggle.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './SearchByToggle.scss';
+
+const SearchByToggle = ({onChange, value}) => {
+
+  // Assign the color to the button according to whether it is
+  // active or not
+  const getClassList = (criteria) => {
+    const color = value === criteria? 'button--red': 'button--grey';
+    return `button button--small ${color}`;
+  }
+
+  return (
+    <div className="search-by-toggle">
+      <span>SEARCH BY:</span>
+      <button className={getClassList('title')}
```

Comment:
If you want to, you can abstract it as another component for reuse. (it's not mandatory, just an idea)